src/org/lumicall/android/preferences/SilentMode.java

```diff
@@ -0,0 +1,302 @@
+package org.lumicall.android.preferences;
+
+import android.app.Activity;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.view.View;
+import android.widget.CheckBox;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import org.lumicall.android.R;
+import org.sipdroid.sipua.ui.Receiver;
+import org.sipdroid.sipua.ui.Settings;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Locale;
+
+/**
+ * Created by urvikagola on 29/1/17.
+ */
+
+public class SilentMode extends Activity {
+    EditText hh1,mm1,hh2,mm2;
```

Comment:
Similar comment to what I said above, hhStart, mmStart, hhEnd, mmEnd would be more legible.